multiopython/_version.py

```diff
@@ -0,0 +1,17 @@
+# file generated by setuptools_scm
+# don't change, don't track in version control
```

Comment:
why is it being added when it says `don't track in version control`? Missing from .gitignore?

Reply:
Whoops